### PR TITLE
adding dbt env command

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds the dbt environment command and its flags to the docs site for the cloud cli.  it also sets it up so that may add more flags/commands if necessary.

it has been added in the following pages:
- https://docs.getdbt.com/reference/dbt-commands -- added to the table list
- https://docs.getdbt.com/reference/commands/dbt-environment -- created a brand new page
- https://docs.getdbt.com/docs/cloud/cloud-cli-installation -- in the installation steps
- https://docs.getdbt.com/docs/cloud/configure-cloud-cli -- in the config steps

PR also renames the sidebar so the cloud cli is less repeated. add;l details [provided here](https://www.notion.so/dbtlabs/dbt-env-show-details-82c33bf72bb34142ac7511ecebcf1b66) on the command and the next steps.

### CODE REVIEW

File: website/docs/reference/commands/dbt-environment.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (account ID, environment ID, environment name, connection type, and more).
+- CAN YOU MAKE CHANGES TO THE ENVIRONMENT?
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]: flagging this question

Comment (cesher) [reply website/docs/reference/commands/dbt-environment.md]: Not currently, we plan to add this functionality at a later date. We shouldn't mention it yet.

**Review by cesher — APPROVED** [review]
Looks great! Thank you!

File: website/docs/reference/commands/dbt-environment.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]:
```suggestion
```

File: website/docs/reference/commands/dbt-environment.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md:9]: @cesher one last question - what is the difference between local config and dbt cloud config in this context? i think i understand the difference in the 'git' context but wanted to clarify this for users. 

is it:
- local config -- details about their local environment setup?
- cloud config - dbt cloud account and connection details

File: website/docs/reference/commands/dbt-environment.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+- Viewing your dbt Cloud configuration details (account ID, environment ID, environment name, connection type, and more).
+
+This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use it, add a command after it like this: `dbt environment [command]`.
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]: i opted for plural here to make it easier for our future selves when more commands/options were added. adding

File: website/docs/reference/commands/dbt-environment.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (account ID, environment ID, environment name, connection type, and more).
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]:
```suggestion
- Viewing your dbt Cloud configuration details (environment ID, environment name, connection type, and more).
```

File: website/docs/reference/commands/dbt-environment.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/reference/commands/dbt-environment.md:9]: going to merge pr and address this when clarified in the shipped channel

File: website/docs/reference/commands/dbt-environment.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+---
+title: "About dbt environment command"
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. The `dbt environment` command is useful for:
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]:
```suggestion
The `dbt environment` command enables you to interact with your dbt Cloud environment. Use the command for:
```

File: website/docs/reference/commands/dbt-environment.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. Use the command for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (environment ID, environment name, connection type, and more).
+
+This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use it, add a command after it like this: `dbt environment [command]`.
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]:
```suggestion
This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use them, add a command or option like this: `dbt environment [command]`.
```

File: website/docs/reference/commands/dbt-environment.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+sidebar_label: "environment"
+id: dbt-environment
+---
+
+The `dbt environment` command enables you to interact with your dbt Cloud environment. Use the command for:
+
+- Viewing your local configuration details (account ID, active project ID, deployment environment, and more).
+- Viewing your dbt Cloud configuration details (environment ID, environment name, connection type, and more).
+
+This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use them, add a command or option like this: `dbt environment [command]`.
```
Comment (mirnawong1) [inline website/docs/reference/commands/dbt-environment.md]:
```suggestion
This guide lists all the commands and options you can use with `dbt environment` in the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). To use them, add a command or option like this: `dbt environment [command]` or use the shorthand  `dbt env [command]`.
```

**Pull request merged**